rest_registration/decorators.py

```diff
@@ -9,7 +9,7 @@ def api_view_serializer_class_getter(serializer_class_getter):
     def _get_serializer_class(self):
         return serializer_class_getter()
 
-    def _get_serializer(self):
+    def _get_serializer(self, instance=None):
```

Comment:
Hi, thank you for the contribution. I have two questions:
1.  Would it be possible to change the code so `_get_serializer` could support more potential cases? For instance, [GenericAPIView.get_serializer](https://github.com/encode/django-rest-framework/blob/cdb8a3c3c86bbb74f17f8880c52000602b015ed4/rest_framework/generics.py#L105-L112) passes the `*args` and `**kwargs` to the serializer class constructor. In other words could you change `_get_serializer` to something like this:

```python
    def _get_serializer(self, *args, **kwargs):
        serializer_class = self.get_serializer_class()
        return serializer_class(*args, **kwargs)
```
2.  Is there an easy way to reproduce this problem with a testcase? I mean some test which for instance creates a renderer which in turn would call the `get_serializer()` method for the profile view (or any other view using `api_view_serializer_class_getter` decorator)?

Reply:
1. That seems to be better, I will do that.
2. I was trying to make test case yesterday, but I was unsuccessful. But you are suggesting much simpler approach, so I will try to make it.

Reply:
@PetrDlouhy Is there any progress on that? If the test is too complicated to do, don't bother doing it; you could just change the `_get_serializer` so it will pass all the data to the serializer class and then I could merge this PR and release next version.